app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/check_your_answers.scala.html

```diff
@@ -36,48 +38,51 @@
     </div>
 }
 
-@tradingStartDate = @{
+
+@tradingStartDate(businessStartDate: BusinessStartDate, businessId: String) = @{
     import implicitDateFormatter.LongDate
     val question: String = messages("check_your_answers.trading_start_date")
-    val answer: String = answers.businessStartDate.startDate.toLocalDate.toLongDate
-    val changeUrl: String = uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.routes.BusinessStartDateController.show(id, isEditMode = true).url
-
+    val answer: String = businessStartDate.startDate.toLocalDate.toLongDate
+    val changeUrl: String = uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.routes.BusinessStartDateController.show(businessId, isEditMode = true).url
```

Comment:
As mentioned above we need to pass in index of the business which we intend to change.